Energy-Saver.Tests/UnitTests.cs

```diff
@@ -0,0 +1,213 @@
+using Energy_Saver.DataSpace;
+using Energy_Saver.Model;
+using Microsoft.EntityFrameworkCore;
+using static Energy_Saver.Model.Serialization;
+
+namespace Energy_Saver.Tests
+{
+    public class UnitTests// : IClassFixture<IndexModel>
+    {
+        //private readonly ILogger<IndexModel> _logger;
+        //private readonly ISuggestionsService _suggestionsService;
+        //private readonly EnergySaverTaxesContext _context;
+
+        //public UnitTests(ILogger<IndexModel> logger, ISuggestionsService suggestionsService, EnergySaverTaxesContext context)
+        //{
+        //    _logger = logger;
+        //    _suggestionsService = suggestionsService;
+        //    _context = context;
+        //}
+
+            [Fact]
+        public async void TaxInputTest()
+        {
+            using (var db = new EnergySaverTaxesContext(Utilities.TestDbContextOptions()))
+            {
+                var taxes = new Taxes
+                {
+                    ID = 1,
+                    UserID = 4,
+                    Year = 2020,
+                    Month = Months.January,
+                    GasAmount = 1,
+                    ElectricityAmount = 2,
+                    WaterAmount = 3,
+                    HeatingAmount = 4
+                };
+
+                List<Taxes> expectedTaxList = new List<Taxes>();
+                expectedTaxList.Add(taxes);
```

Comment:
Could try adding in more than one entry.

Reply:
added 2 more